docs/source/learn/core_notebooks/pymc_aesara.ipynb

```diff
@@ -0,0 +1,2039 @@
+{
```

Comment:
<p>Do we need this? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc/pull/5721/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
In a previous comment I thought you also wanted to compare the owners. I do believe is a bit redundant so if you want I can remove it.

Reply:
<p>In a previous comment I thought you also wanted to compare the owners. I do believe is a bit redundant so if you want I can remove it.</p>